api/services/ocean/spark/schemas/oceanSparkApplicationTrendBucket.yaml

```diff
@@ -0,0 +1,41 @@
+type: object
+required:
+- start
+- end
+- completed
+- failed
+- killed
+- timed_out
+description: >
+  Sum of application states for apps that ended in a given time range.
```

Comment:
The number of applications that ended in a given time range, grouped by the terminal state they reached.